### DISCUSSION TITLE
Is it possible to visualize the result of equalizer? What does "calculate frequencies" button in equalizer do?

### DISCUSSION THREAD

**Discussion by greennerve** [wwmm/easyeffects#1925] [Q&A]
It's difficult to adjust each band and the its Q value without visualizing the results. I'd like to know if easyeffects has already such function like in [this graph](https://imgur.com/a/V1PXrBq). Another question, what does "calculate frequencies" button do in equalizer?

**Comment by wwmm** [comment]
Unfortunately this is a little hard to do at this moment #515. Nowadays we have our own LV2 wrapper. But it is still very complicated to get the graph when using third party plugins.

**Reply by wwmm** [reply]
Something you may try is loading the Linux Studio Plugins Equalizer in a LV2 plugin host so you can use its native window. I think some people do that to see the plots and export a APO preset in the end.


>what does "calculate frequencies" button do in equalizer?

It tries to set the bands in a way that mimics a graphical equalizer with the chosen number of bands. It isn't really the same thing as using a proper graphical equalizer. But integrating in EasyEffects tons of different graphical equalizers isn't the most viable thing to do.
